src/createQueryGroup.ts

```diff
@@ -17,13 +17,13 @@ export type QueryGroup<
   active: boolean,
 }
 
-export type QueryGroupOptions = {
+export type QueryGroupOptions = QueryOptions<any> & {
```

Comment:
This doesn't seem correct to me, it doesn't accept all the options. Just retries and maybe onDispose?